src/init.cpp

```diff
@@ -1509,7 +1507,7 @@ bool AppInit2(boost::thread_group& threadGroup, CScheduler& scheduler)
     CAutoFile est_filein(fopen(est_path.string().c_str(), "rb"), SER_DISK, CLIENT_VERSION);
     // Allowed to fail as this file IS missing on first startup.
     if (!est_filein.IsNull())
-        mempool.ReadFeeEstimates(est_filein);
+        mempool.ReadFeeEstimates(est_filein, fFeeEstimatesInitialized);
```

Comment:
`fFeeEstimatesInitialized` is never set to `true` before this point, and `CTxMemPool::ReadFeeEstimates` is not called in `zcashd` anywhere else, so revert the addition of this parameter to `CTxMemPool::ReadFeeEstimates`.